pkg/virt-api/webhooks/validating-webhook/admitters/vms-admitter.go

```diff
@@ -454,6 +454,14 @@ func validateLiveUpdateFeatures(field *k8sfield.Path, spec *v1.VirtualMachineSpe
 		causes = append(causes, validateLiveUpdateMemory(field, &spec.Template.Spec.Domain, spec.Template.Spec.Architecture)...)
 	}
 
+	if spec.UpdateVolumesStrategy != nil && !config.VolumesUpdateStrategyEnabled() {
+		causes = append(causes, metav1.StatusCause{
+			Type:    metav1.CauseTypeFieldValueInvalid,
+			Message: fmt.Sprintf("%s feature gate is not enabled in kubevirt-config", virtconfig.VolumesUpdateStrategy),
+			Field:   "updateVolumesStrategy",
+		})
+	}
+
```

Comment:
Not now but maybe in the future, would it make sense to add a warning here if a volume associated with a DataVolumeTemplate was changed?

Reply:
yes, definetly!

Reply:
Actually, I think this is already broken today. We can already change the volumes after a restart. However, if we replace one and leave the data volumes this could can problems. I have only experience this with volume migration, but it should happen also in this case.
I will try to reproduce the issue and address this in another PR.

Reply:
In order to apply the new volume, the user needs to remove the old datavolumetemplate of the update volume

Reply:
I see, makes sense. So this case is already covered.